### PR TITLE
Add validation for missing file(s) in .dockstore.yml

### PR DESCRIPTION
**Pull request by kathy-t**
For #3007 

If a file in .dockstore.yml doesn't exist in the GitHub repo, a validation warning will appear. This is what it looks like right now with the backend change:
Screenshot from 2020-03-05 14-42-31

There are some changes that need to be made in the UI, but I'll do that after this PR.

### CODE REVIEW

**Comment by denis-yuen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- dockstore-webservice/src/main/java/io/dockstore/webservice/languages/ServicePrototypePlugin.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dockstore/dockstore/pullRequest?prid=5130068&bid=16858804)

File: dockstore-webservice/src/main/java/io/dockstore/webservice/languages/ServicePrototypePlugin.java [hunk 1]
```diff
@@ -63,6 +64,19 @@ public VersionTypeValidation validateWorkflowSet(String initialPath, String cont
                 if (files == null) {
                     validationMessageObject.put(initialPath, "The key 'files' does not exist.");
                     isValid = false;
+                } else {
+                    // check that files in .dockstore.yml exist
+                    ArrayList<String> missingFiles = new ArrayList<String>();
```
Comment (coverbeck) [inline dockstore-webservice/src/main/java/io/dockstore/webservice/languages/ServicePrototypePlugin.java]: If so inclined, you could rewrite this and the for loop with map and filter. But it's fine the way it is.

Feel free to ignore this; I'm just personally liking the FP style more.

Comment (denis-yuen) [inline dockstore-webservice/src/main/java/io/dockstore/webservice/languages/ServicePrototypePlugin.java]: Use List on the left unless you really rely on ArrayList
https://java2blog.com/java-list-vs-arraylist/

**Comment by denis-yuen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- dockstore-webservice/src/main/java/io/dockstore/webservice/languages/ServicePrototypePlugin.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dockstore/dockstore/pullRequest?prid=5130068&bid=16858804)

**Comment by denis-yuen** [comment]
Seeing a bunch of postgres failures here too, attention @coverbeck 
Example https://travis-ci.org/dockstore/dockstore/jobs/659166412?utm_medium=notification&utm_source=github_status

**Comment by denis-yuen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- dockstore-webservice/src/main/java/io/dockstore/webservice/languages/ServicePrototypePlugin.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dockstore/dockstore/pullRequest?prid=5130068&bid=16858804)

**Pull request merged**